pkgs/top-level/perl-packages.nix

```diff
@@ -16969,10 +16969,10 @@ let
 
   PerlMagick = buildPerlPackage rec {
```

Comment:
```suggestion
  Image-Magick = PerlMagick; # added 2021-08-02
  PerlMagick = buildPerlPackage rec {
```

Reply:
yeah im renaming rn

Reply:
Since the distribution is called to `Image-Magick` i think we should rename the attribute to `ImageMagick` (without the hyphen) to match the conventions elsewhere in perl-packages, and alias `PerlMagick` to it.